src/main/java/dev/hephaestus/atmosfera/client/sound/AtmosphericSoundHandler.java

```diff
@@ -104,11 +105,11 @@ public MusicSound getMusicSound(MusicSound defaultSound) {
                 float volume = definition.getVolume(world);
 
                 if (volume > 0.0125) {
-                    int weight = Objects.requireNonNull(soundManager.get(definition.soundEvent().getId())).getWeight();
+                    int weight = Objects.requireNonNull(soundManager.get(definition.soundId())).getWeight();
 
                     sounds.add(new Pair<>(weight, MUSIC.computeIfAbsent(definition, id -> {
                         Atmosfera.debug("createIngameMusic: {}", definition.id());
-                        return MusicType.createIngameMusic(definition.soundEvent());
+                        return MusicType.createIngameMusic(new SoundEvent(definition.soundId()));
```

Comment:
See `AtmosphericSoundSerializer` below, this is where we create our sound event (vanilla only uses its id here).